src/test/java/uk/gov/hmcts/cmc/claimstore/utils/MonthIncrementTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.cmc.claimstore.utils;
+
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MonthIncrementTest {
```

Comment:
is it CalculateMonthIncrementTest? Its must to name the Test class prefixed with the classname, thus the IDE can easily identify tests.